<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleFilterDialog</class>
 <widget class="QDialog" name="SimpleFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>276</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Trace Filter</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>70</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mnemonic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>260</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Instruction</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_min_va">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Min. VA</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_max_va">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Max. VA</string>
   </property>
  </widget>
  <widget class="QLabel" name="va_range_dash">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_mnemonic</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SimpleFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SimpleFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
